README.md

```diff
@@ -190,6 +188,22 @@ $ cargo run --release --features "critical-section-impl,rt,defmt" --example pwm_
 (The `pwm_blink` example doesn't need all these feature flags. They are listed here
 so you can use the same command for all examples.)
 
+If you want to create a UF2 file, which is loaded by copying it over to the
+RPI-RP2 mass storage device, use the `picotool uf2 convert` command on your
+compiled program with the `-t elf` argument.
+
+```console
+$ picotool uf2 convert target/thumbv6m-none-eabi/release/pwm_blink -t elf
```

Comment:
```suggestion
$ picotool uf2 convert -t elf target/thumbv6m-none-eabi/release/pwm_blink
```

In other cases, we put the `-t elf` before the file name. Just wanted to be consistent in case it causes confusion (it doesn't really matter where you put it).